modules/sagemaker/custom-kernel/scripts/build_docker.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com
+echo Building the Docker image...
+cache_image=$IMAGE_URI
```

Comment:
Most likely there will be no caching due to CodeBuild compute instance...but this is ok to leave

Reply:
The caching is there to reuse image that is already built. For example if you add new layer to docker image, you will reuse the previously built layers

Reply:
Unfortunately, the Codebuild Compute and storage is ephemeral...so there really is no cache.  But, it won't hurt to keep this.....

Reply:
It does not depend on codebuild storage. It pulls the cache from ECR (line below).